### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by ksigei**
### In this milestone, I was able to:

- Set up a new project with webpack.
- Create an index.html file and set it as a template using the HTML Webpack Plugin. Create an empty To Do List placeholder (<div> or <ul> element).
- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object  contains three keys:
-         description [string].
-         completed [bool].
-         index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
-Make sure on page load render the dynamically created list of tasks in the dedicated placeholder. The list then appears in order of the index values for each task.
- Create a style.css by using Webpack Style/CSS Loader and set rules for the To Do List.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+          <input class="checkbox" type="checkbox">
+            Study CSS
+          <button class="delete-button">X</button>
+        </li>
+        <li class="item" data-key="1594003133171">
+          <input class="checkbox" type="checkbox">
+            Watch football
+          <button class="delete-button">X</button>
+        </li>
+      </ul>
```
Comment (ShoiraTa) [inline dist/index.html:36]: - [ ] These tasks should be rendered from JS as a part of dynamic content. Please make sure that JS has an array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. Eg.:

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 1,
  },
  {
    description: 'An example description in string',
    completed: false,
    index: 2,
  }
];
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ksigei ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

- [ ] Please update the README file and add all the relevant sections like:
- Description
- Built With
- Getting started, 
- Author
- Acknowledgments
- License


- [ ] Check more comments under the review.

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (ksigei) [reply dist/index.html:36]: Okay, let me implement it

**Review by mikethreels — APPROVED** [review]
Hi Kip, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**